stacks/core/README.md

```diff
@@ -33,12 +46,17 @@ This stack will be deployed with the basic services, which are related to the th
    knot-cloud create-user <email> <password> --server api.knot.cloud --protocol https
    ```
 
-1. Create a new user account in the **KNoT Fog** and save the output:
-
    ```bash
-   knot-cloud create-user <email> <password> --server api.fog --protocol http
+   knot-cloud create-user knot@cesar.org.br exemplo-senha --server api.knot.cloud --protocol https
+
+   user successfully created
+   {
+     "token": "some-token-returned"
+   }
    ```
 
+1. Create a new user account in the **KNoT Fog** by repeating the same previous command with the options `--server api.fog` and `--protocol http` and save the output. The same e-mail and password can be used.
```

Comment:
@kbcvcbk is it enough?
Another option is:
"Repeat the previous command by replacing the options with `--server api.fog` and `--protocol http` to create a new user account in the KNoT Fog and save the output. The same e-mail and password can be used."